bigbone/src/main/kotlin/social/bigbone/api/entity/Report.kt

```diff
@@ -17,5 +17,5 @@ data class Report(
      * Whether an action was taken yet.
      */
     @SerializedName("action_taken")
-    val actionTaken: String = ""
+    val actionTaken: String = "" // todo: Change this to type Boolean
```

Comment:
Open TODO comment: I'd suggest that we change type to `Boolean` as you suggested.

Reply:
This is already changed in #277 but I can change it to Boolean here to reduce merge conflicts. 👍

Reply:
No worry, you can leave it as it is.

Reply:
Done 👍

Reply:
Also good :-)